.github/workflows/release.yml

```diff
@@ -26,51 +26,12 @@ jobs:
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
           go get -v -t -d ./...
-      - name: Build govvv
+      - name: Build release artifacts
         run: |
-          go build -modfile .bingo/govvv.mod -o=. github.com/ahmetb/govvv
-      - name: Build gox
-        run: |
-          go build -modfile .bingo/gox.mod -o=. github.com/mitchellh/gox
-      - name: Compile
-        run: |
-          VERSION=${GITHUB_REF##*/}
-          export PATH=${PATH}:`go env GOPATH`/bin
-          ./gox -osarch="linux/amd64 linux/386 linux/arm darwin/amd64 windows/amd64" -output="hub-{{.OS}}-{{.Arch}}" -ldflags="$(./govvv -flags -version ${VERSION} -pkg $(go list ./buildinfo))" ./cmd/hub
-          ./gox -osarch="linux/amd64 linux/386 linux/arm darwin/amd64 windows/amd64" -output="buck-{{.OS}}-{{.Arch}}" ./cmd/buck
-      - name: Collect artifacts
-        run: |
-          VERSION=${GITHUB_REF##*/}
-          OUT=release/cli
-          mkdir -p ${OUT}
-          mkdir -p tmp
-          cp LICENSE tmp/
-          cp README.md tmp/
-          cp dist/install tmp/
-          cd tmp
-          declare -a arr=("darwin-amd64" "windows-amd64.exe" "linux-amd64" "linux-386" "linux-arm")
-          for i in "${arr[@]}"
-          do
-              OSARCH=${i%.*}
-              EXT=$([[ "$i" = *.* ]] && echo ".${i##*.}" || echo '')
-              cp ../hub-${i} hub${EXT}
-              cp ../buck-${i} buck${EXT}
-              if [ "${EXT}" == ".exe" ]; then
-                  zip hub_${VERSION}_${OSARCH}.zip LICENSE README.md hub${EXT}
-                  zip buck_${VERSION}_${OSARCH}.zip LICENSE README.md buck${EXT}
-                  mv hub_${VERSION}_${OSARCH}.zip ../${OUT}/
-                  mv buck_${VERSION}_${OSARCH}.zip ../${OUT}/
-              else
-                  tar -czvf hub_${VERSION}_${OSARCH}.tar.gz LICENSE README.md install hub
-                  tar -czvf buck_${VERSION}_${OSARCH}.tar.gz LICENSE README.md install buck
-                  mv hub_${VERSION}_${OSARCH}.tar.gz ../${OUT}/
-                  mv buck_${VERSION}_${OSARCH}.tar.gz ../${OUT}/
-              fi
-          done
-          cd .. && rm -rf tmp
-          echo $(ls ./release/cli)
```

Comment:
All this logic lives in `Makefile` now and is a more simple and reusable function to release any package as an artifact